proc/variables.go

```diff
@@ -29,16 +31,24 @@ const (
 type Variable struct {
 	Addr      uintptr
 	Name      string
-	Value     string
-	Type      string
-	dwarfType dwarf.Type
+	DwarfType dwarf.Type
+	RealType  dwarf.Type
+	Kind      reflect.Kind
 	thread    *Thread
 
-	Len       int64
-	Cap       int64
+	Value constant.Value
+
+	Len int64
```

Comment:
These fields are fairly obvious but please add some comments with some docs explaining these fields and when they are valid, etc.